bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -486,6 +486,14 @@ bool PreProcessor::handlePossiblyExpiredRequest(const RequestStateSharedPtr &req
     const auto &clientId = reqStatePtr->getClientId();
     const auto &batchCid = reqStatePtr->getBatchCid();
     const auto &reqCid = reqStatePtr->getReqCid();
+    if (reqStatePtr->getPreProcessRequest() == nullptr) {
```

Comment:
if (reqStatePtr->getPreProcessRequest()) is enough

Reply:
ok, i will change it